### ISSUE TITLE
Animation smoothing - Phoenix pet

### ISSUE THREAD

**Issue by craycrayxx** [runelite/runelite#14141]
When having your Phoenix pet out with animation smoothing it freezes for a second every time the flying animation ends and restarts.

**Comment by rarick** [comment]
The animation smoothing issues have been bothering me too, going to see if I can fix it.

It's caused by the last frame of the animation not interpolating to the first frame.

**Comment by rarick** [comment]
I've contacted the admins on discord and discussed it with them. There's no interpolation smoothing between the last and first frames because of things such as windmills and waterwheels which rotate some amount, and then return to their starting state.

If interpolation were done on these animations, the objects would appear to rotate back and forth repeatedly, which would be jarring.

@Adam- suggested it could be possible to blocklist animations which needed to not interpolate (such as the windmills and waterwheels mentioned above). I offered that we could search for animations which are solely rotations, and potentially generate a blocklist that way.

@abextm searched the animations, and identified 423 animations which are "pure rotations," which are candidates for being in the blocklist.

I am now discussing if it's possible for this to be implemented, and am willing to assist.
